src/App.tsx

```diff
@@ -85,8 +85,9 @@ const App = () => {
           />
           <Stack.Screen name="Redux" component={ReduxScreen} />
           <Stack.Screen name="EndToEndTests" component={EndToEndTestsScreen} />
-          <Stack.Screen name="ToolStore" component={ToolStore} />
+          <Stack.Screen name="EmpowerPlant" component={EmpowerPlant} />
```

Comment:
@thinkocapo - I believe we want to call this `products`?

Reply:
approved outside of this comment. Fantastic work 💯

Reply:
I think you're right, otherwise you'd never see a Products transaction on this performance page view:
https://sentry.io/organizations/testorg-az/performance/?project=5716557&statsPeriod=24h

But if we rename it to products, then we'll never see it saying EmpowerPlant in performance page anywhere, because **there's no homepage (screen)**. You press the button for Empower Plant' from the list app and it takes you directly to the products page.....are we okay with this? I'm okay with it.

I didn't want too many screens to click through, and the images we have on the web app EmpowerPlant homepage are much too large for any home-screen for EmpowerPlant in react native. I'm okay with going right to products, it's faster.

@cstavitsky @ndmanvar thoughts

Reply:
fixed